pages/content/ch3/sets.mdx

```diff
@@ -0,0 +1,80 @@
+import { Callout, Tabs} from 'nextra/components'
+import Problem from '@/components/problem'
+
+# The most basic object
+
+In mathematics a set constitutes the most basic object. We define
+a mathematical object by the properties of the operations we can perform on it.
+
+In the case of sets, we only have a single operation: the membership predicate taking two
+sets and mapping them to a truth value. In infix notation:
+$$
+x \in A \\
+x \notin A \iff \neg (x \in A)
+$$
+
+We can write a set by enumerating its elements:
+$$
+\{ A, B, C \}
+$$
+
+<Callout type="warning" emoji="💡">
+  When enumerating sets, the order of elements does not matter. Elements
+  can be repeated without an effect:
+    $$ \{ A, B, C \} = \{ A, C, B \} = \{ A, B, C, A \} $$
+</Callout>
+
+The membership predicate has the following properties (simplified version of the *ZF axioms*).
+Note that these
+axioms define the existence of special sets while also introducing some shorthand notation.
+
+<Callout type='axiom'>
+1. **Empty set**: There exists an empty set, usually denoted as $\emptyset$.
+    $$
+    \forall x \neg (x \in \emptyset)
+    $$
+2. **Equality**:
+    $$
+    \begin{align*}
+    A = B &\iff \forall x (x \in A \leftrightarrow x \in B) \\
+            &\iff A \subseteq B \land B \subseteq A
+    \end{align*}
+    $$
+3. **Subset**:
+    $$
+    \begin{align*}
+    A \subseteq B &\iff \forall x (x \in A \to x \in B) \\
+    A \subset B &\iff A \subseteq B \land A \neq B \\
+    \end{align*}
+    $$
+3. **Axiom of pairing**: For any sets $A, B$ there exists a set $\{A, B\}$.
+4. **Union**: For any sets $A, B$ there exists a set $A \cup B$. Defined as:
+    $$
+    x \in A \cup B \iff x \in A \lor x \in B
+    $$
+5. **Intersection**: For any sets $A, B$ there exists a set $A \cap B$. Defined as:
+    $$
+    x \in A \cap B \iff x \in A \land x \in B
+    $$
+6. **Difference**: For any sets $A, B$ there exists a set $A \setminus B$. Defined as:
+    $$
+    x \in A \setminus B \iff x \in A \land x \notin B
+    $$
+5. **Infinity**: There exists a non-finite set of natural numbers $\mathbb{N}$ as defined in
+chapter 3.2.7 of the script.
+6. **Filter**: For any predicate $P$ and set $A$ there exists a set containing all
+the elements of $A$ for which $P$ holds. In set notation:
+    $$
+    \{ x \in A \mid P(x) \}
+    $$
+7.  **Power set**: For any set $A$ there exists a set $\mathcal{P}(A)$ containing all subsets of $A$.
+    $$
+    x \in \mathcal{P}(A) \iff x \subseteq A
+    $$
+</Callout>
+
+<Callout type="info" emoji="💡">
+  The existence of a singleton set $\{x\}$ is a direct consequence of the axiom of pairing.
```

Comment:
Das waere eine (einfache) Aufgabe; beweise dass $\{x\}$ existiert (gegeben Axiom of Pairing)